### PR TITLE
Update to Registry 0.0.3

### PR DESCRIPTION
**Pull request by pelle**
Due to the change to bytes32 for type and names this introduced a new simpler naming standard.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,8 +96,10 @@ event DIDDelegateChanged(
 
 The only 2 delegateTypes that are currently published in the DID Document are:
 
-- `Secp256k1VerificationKey2018` which is added to the `publicKey` section of document
-- `Secp256k1SignatureAuthentication2018` which is also added to the `publicKey` section of document. An entry is also added to the `authentication` section of document
+- `veriKey` Which adds a `Secp256k1VerificationKey2018` to the `publicKey` section of document
+- `sigAuth` Which adds a `Secp256k1SignatureAuthentication2018` to the `publicKey` section of document. An entry is also added to the `authentication` section of document.
```
[the remark this answers is not in the document]
Comment (pelle) [reply README.md]: There is something to be said for human readable names

File: README.md [hunk 2]
```diff
@@ -108,28 +110,70 @@ Non ethereum keys, service elements etc can be added using attributes. Attribute
...
+'''
+
+#### Base64 encoded Ed25519 Verification Key
+
+A `DIDAttributeChanged` event for the identity `0xf3beac30c498d9e26865f34fcaa57dbb935b0d74` with the name `did/pub/Ed25519/veriKey/base64` and the value of `0xb97c30de767f084ce3080168ee293053ba33b235d7116a3263d29f1450936b71` generates a `PublicKey` entry like this:
+
+'''js
+{
+  id: "did:ethr:0xf3beac30c498d9e26865f34fcaa57dbb935b0d74#delegate-1",
+  type: "Secp256k1VerificationKey2018",
```
[the remark this answers is not in the document]
Comment (pelle) [reply README.md]: @oed Thats not the same. This is specifically for the signing curve.

[the remark this answers is not in the document]
Comment (oed) [reply README.md]: Ok, I thought we where only using `Secp256k1` for signing.

File: README.md [hunk 3]
```diff
@@ -108,28 +110,70 @@ Non ethereum keys, service elements etc can be added using attributes. Attribute
...
-|section|type|encoding|
-|-------|----|--------|
-|`publicKey`| Any valid Public Key type eg. `Secp256k1VerificationKey2018`, `Ed25519VerificationKey2018`, `RsaVerificationKey2018` | `publicKeyHex` (default), `publicKeyBase64` (please submit PRs for `publicKeyPem`, `publicKeyJwk`, `publicKeyBase58`)|
-|`service`| Any valid service type eg `HubService`, `AgentService` | n/a |
+#### Hex encoded Secp256k1 Verification Key
 
-Values should be encoded in binary bytes for efficiency reasons. Encoding in the DID document will be converted according to method. Any unsupported attributes and unknown encodings will be ignored.
+A `DIDAttributeChanged` event for the identity `0xf3beac30c498d9e26865f34fcaa57dbb935b0d74` with the name `did/pub/Secp256k1/veriKey/hex` and the value of `0x02b97c30de767f084ce3080168ee293053ba33b235d7116a3263d29f1450936b71` generates a `PublicKey` entry like this:
+
+'''js
```
Comment (oed) [inline README.md]: Here (and some more places) `'''` is being used instead of ``` which causes some problems with formatting.

**Pull request merged**